helm/mediator/templates/deployment.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.deployment_name }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: {{ .Values.deployment_name }}
+spec:
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      app: {{ .Values.deployment_name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.deployment_name }}
+    spec:
+      containers:
+      - name: {{ .Values.container.name }}
+        image: " {{ .Values.image.name }}:{{ .Values.image.tag }} "
```

Comment:
The mediator needs a caches directory, I'm not 100% sure why but it does. Let's add that as an emptyDir:

```yaml
      volumes:
        - name: cache-volume
          emptyDir:
            sizeLimit: 1Gi
```

and 

```yaml
          volumeMounts:
            - name: cache-volume
              mountPath: /.cache
```